### PR TITLE
name and iconography change from ninja to stars

### PR DESCRIPTION
**Pull request by matallen**

### CODE REVIEW

File: src/main/webapp/mojo/mojo-dashboard-card.jsp [hunk 1]
```diff
@@ -356,7 +357,9 @@ function _displ(username, dontReset){
 			    
 		    	
 		    	if (key=="level"){
-						document.getElementsByClassName("_level")[0].src=ctx+"/../../images/"+value.toLowerCase()+"_belt_icon.png";
+						//document.getElementsByClassName("_level")[0].src=ctx+"/../../images/"+value.toLowerCase()+"_belt_icon.png";
```
Comment (sabre1041) [inline src/main/webapp/mojo/mojo-dashboard-card.jsp:360]: Remove the comments?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Small changes. Otherwise good

File: src/main/webapp/mojo/mojo-dashboard-card.jsp [hunk 2]
```diff
@@ -267,6 +267,7 @@ Utils = {
 }
 
 function getUsername(){
+	return "ablock";
```
[the remark this answers is not in the document]
Comment (matallen) [reply src/main/webapp/mojo/mojo-dashboard-card.jsp:270]: wow, thanks for spotting that - you were my test case today!

File: src/main/webapp/mojo/mojo-dashboard-card.jsp (hunk 1 above)
Comment (matallen) [reply src/main/webapp/mojo/mojo-dashboard-card.jsp]: removed

File: src/main/webapp/mojo/mojo-dashboard-card.jsp (hunk 2 above)
[the remark this answers is not in the document]
Comment (matallen) [reply src/main/webapp/mojo/mojo-dashboard-card.jsp]: this has been removed in the commit below

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**